src/pages/users/users-details/users-details.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import { Grid, Button } from '@material-ui/core';
+import { withRouter } from 'react-router';
+import { useStyles } from './users-details.styles';
+import useUsersHandler from '../../../utils/use-users-handlers';
+import LoadingBar from '../../../components/loading-bar';
+import { config } from '../../../configs';
+import { updateUserStatus } from '../../../redux/users/users.actions';
+import { closeDialog } from '../../../redux/dialog-window/dialog-window.actions';
+import useSuccessSnackbar from '../../../utils/use-success-snackbar';
+import UserDetailsHeader from './containers/user-details-header';
+import UserDetailsBody from './containers/user-details-body';
+import UserDetailsSection from './containers/user-details-section';
+import UserInput from './containers/user-details-input';
+
+const {
+  USER_ACTIVE_TITLE,
+  USER_UNACTIVE_TITLE,
+  SWITCH_USER_STATUS_TITLE
+} = config.buttonTitles;
+const { USER_ACTIVE_STATUS, USER_UNACTIVE_STATUS } = config.statuses;
+const { SWITCH_USER_STATUS_MESSAGE } = config.messages;
+
+const UsersDetails = (props) => {
+  const { match } = props;
+
+  const { openSuccessSnackbar } = useSuccessSnackbar();
+  const styles = useStyles();
+  const dispatch = useDispatch();
+
+  const { loading } = useSelector(({ Users }) => ({
+    loading: Users.usersLoading
+  }));
+
+  const { id } = match.params;
+
+  const {
+    firstName,
+    lastName,
+    country,
+    city,
+    adress,
+    postCode,
+    isBanned
+  } = useUsersHandler(id);
+
+  const status = isBanned ? USER_UNACTIVE_STATUS : USER_ACTIVE_STATUS;
```

Comment:
`USER_UNACTIVE_STATUS` should be `USER_INACTIVE_STATUS`

Reply:
fixed this one